dpctl/tensor/_manipulation_functions.py

```diff
@@ -0,0 +1,63 @@
+#                       Data Parallel Control (dpctl)
+#
+#  Copyright 2020-2021 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import numpy as np
+
+import dpctl.tensor as dpt
+
+
+def _check_value_of_axes(axes):
+    check_array = np.zeros(len(axes))
+    for i in axes:
+        ii = i.__index__()
+        if ii < 0 or ii > len(axes) or check_array[ii] != 0:
+            return False
+        check_array[ii] = 1
+    del check_array
```

Comment:
Since `check_array` is a temporary, it is going to be collected after the next instruction (`return`). It does not hurt anything either though.